session_0/fizz_buzz.rb

```diff
@@ -29,13 +29,26 @@
 # For example, fizz_buzz(n: 6, x: 2, y: 3) should return the following:
 # ["1", "Fizz", "Buzz", "Fizz", "5", "FizzBuzz"]
 #
-# What happens if the function is called by x (or y) = 0 or negative? 
+# What happens if the function is called by x (or y) = 0 or negative?
```

Comment:
Please don't push session 0, 1, 2 and 3 again

Reply:
How do I stop that? I just checkout to a different branch and do `git push`, and all of this gets pushed too

Reply:
Check out your branches off main instead of whatever is your last working branch because the last working branch contains earlier sessions